subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/maven/MavenPomPackagingResolveIntegrationTest.groovy

```diff
@@ -333,4 +333,27 @@ compile('group:projectA:1.0') {
         succeeds 'retrieve'
     }
 
+    @Issue('https://github.com/gradle/gradle/issues/1789')
+    def "ignore artifact when relocation is defined in pom"() {
+        given:
+        buildFile << """
+repositories { jcenter() }
+configurations { compile }
+dependencies {
+    // http://repo1.maven.org/maven2/org/apache/commons/commons-io/1.3.2/commons-io-1.3.2.pom
+    compile 'org.apache.commons:commons-io:1.3.2' 
```

Comment:
You're right, I should recreate this scenario. It might be a bit hard but I'm willing to have a try.